<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Employee Accounts</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/svg/grey/account-multiple.svg</normaloff>:/svg/grey/account-multiple.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="fastSearchEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Fast search</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QTableView" name="employeeTableView">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>170</width>
              <height>0</height>
             </size>
            </property>
            <property name="title">
             <string>Employee</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="addEmployeeButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Add </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="editEmployeeButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>Edit  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="removeEmployeeButton">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>     Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QGroupBox" name="queryShortcutsBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="title">
             <string>Query shortcuts</string>
            </property>
            <layout class="QVBoxLayout" name="sqlShortcutLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMaximumSize</enum>
             </property>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Filte&amp;rs</string>
    </property>
    <addaction name="manageFiltersAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Settin&amp;gs</string>
    </property>
    <addaction name="actionNationality"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionLicenses"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilters"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/svg/grey/logout.svg</normaloff>
     <activeoff>:/svg/white/logout.svg</activeoff>
     <activeon>:/svg/white/logout.svg</activeon>:/svg/grey/logout.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="manageFiltersAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/svg/grey/filter.svg</normaloff>
     <normalon>:/svg/grey/filter.svg</normalon>
     <activeoff>:/svg/white/filter.svg</activeoff>
     <activeon>:/svg/white/filter.svg</activeon>:/svg/grey/filter.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Manage filters</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionNationality">
   <property name="text">
    <string>&amp;Nationality</string>
   </property>
  </action>
  <action name="actiontest3">
   <property name="text">
    <string>t&amp;est3</string>
   </property>
  </action>
  <action name="actiontest5">
   <property name="text">
    <string>te&amp;st5</string>
   </property>
  </action>
  <action name="actiontest6">
   <property name="text">
    <string>test&amp;6</string>
   </property>
  </action>
  <action name="actiontest3_2">
   <property name="text">
    <string>test&amp;3</string>
   </property>
  </action>
  <action name="actiontest4">
   <property name="text">
    <string>test&amp;4</string>
   </property>
  </action>
  <action name="actionnote">
   <property name="text">
    <string>&amp;note</string>
   </property>
  </action>
  <action name="actionLicenses">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/svg/grey/copyright.svg</normaloff>
     <activeoff>:/svg/white/copyright.svg</activeoff>
     <activeon>:/svg/white/copyright.svg</activeon>:/svg/grey/copyright.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Licences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addEmployeeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showEmployeeDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>840</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>797</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNationality</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNationalityDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeEmployeeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeSelectedEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>797</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>editEmployeeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>editSelectedEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>758</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>754</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manageFiltersAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFiltersDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showFiltersDialog()</slot>
  <slot>showEmployeeDialog()</slot>
  <slot>showNationalityDialog()</slot>
  <slot>removeSelectedEmployee()</slot>
  <slot>editSelectedEmployee()</slot>
 </slots>
</ui>
